mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -49,5 +50,14 @@ impl BridgeController {
                 error!(message = "error while running bridge", err = %e);
             }
         }
+
+        // TODO: bridge controller will eventually listen for updates via the twin
```

Comment:
I'm not sure why we need to hand controller indefinitely.  shouldn't it be stopped once all bridges finish their work?

Reply:
I was thinking that the bridge controller should always be listening to updates. It might need to start a remote bridge eventually so it should not stop.

Reply:
Since shutdown and the twin update listening are both not implemented yet, I felt this was a good short term solution. Once shutdown is implemented it will be able to cancel the twin update listening and shut down.